src/components/LessonInfoDisplay/LessonInfoDisplay.js

```diff
@@ -37,6 +43,17 @@ function LessonInfoDisplay(props) {
     );
   }
 
+  function handleDeleteLessonConfirmation(isVisible) {
+    setShowDelete(isVisible);
+  }
```

Comment:
if this essentially is just renaming `setShowDelete`, why don't you just use `setShowDelete` instead of making a new function? & the only time that this is used is setting it to true, so it would have been better as `setShowDelete(true)`

Reply:
ope you're 100% correct, I'll fix this haha